sdk/loadtesting/load-testing-rest/src/models.ts

```diff
@@ -220,146 +229,16 @@ export interface ResourceMetric {
 }
 
 /** Load test run model */
-export interface TestRun {
```

Comment:
@catalinaperalta Do you know these columns are removed?

Reply:
Pushed a fix to my branch.